spec/commands/get_user_pull_request_metrics_spec.rb

```diff
@@ -34,14 +34,25 @@ def perform(*_args)
     end
 
     context "with valid date limit" do
+      let!(:valid_review) do
+        create(
+          :pull_request_review,
+          github_user: github_user,
+          pull_request: valid_pull_request,
+          created_at: Time.zone.now - 2.minutes,
+          approved_at: Time.zone.now - 1.minute
+        )
+      end
+
       it "return correct information" do
         expect(perform(github_user: github_user, limit_month: 9)).to eq(
-          "pull_requests_information" => { valid_pull_request.id => {
-            pr_created_at: valid_pull_request.gh_created_at.to_s,
-            pr_title: "Prueba",
+          pull_requests_information: { valid_pull_request.id => {
+            pr_created_at: valid_pull_request.gh_created_at.to_s, pr_title: "Prueba",
+            first_response: valid_review.created_at.to_s,
+            last_approval: valid_review.approved_at.to_s,
             review_request_created_at: valid_review_request.created_at.to_s,
-            time_delta: 120
-          } }, "personal_mean" => 120
+            pr_merget_at: valid_pull_request.gh_merged_at.to_s
+            } }
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.